### ISSUE TITLE
[Bug]: No axsis options when using modern Xbox controller 

### ISSUE THREAD

**Issue by SoulLad** [emoose/OutRun2006Tweaks#124]
### Tweaks version

Tweaks v0.5.1 - bugfixes ahoy!

### Describe your issue here (please attach the OutRun2006Tweaks.log file with your report, you can drag+drop it into the text area)

cant set triggers as analog inputs ingame
[OutRun2006Tweaks.log](https://github.com/user-attachments/files/16736767/OutRun2006Tweaks.log)

**Comment by emoose** [comment]
Which OS do you run on?
Maybe Xidi can help with it, you can change how triggers/etc get mapped: https://github.com/samuelgr/Xidi/releases/tag/v4.2.0

If you rename the outrun2006tweaks dinput8.dll to winmm.dll, you can extract dinput8.dll from xidi zip and use the xidi.ini from https://github.com/emoose/OutRun2006Tweaks/issues/7#issuecomment-2166208309

It's been mixed whether that actually helps or not though, maybe need to look into creating a custom xidi mapping for C2C some time.

**Comment by SoulLad** [comment]
Windows 11, till give this a try. it seems the game wont recognize the controller as an analog input, kinda like it thinks its a ps2 controller but the name still says "xbox one"

**Comment by SoulLad** [comment]
> Windows 11, till give this a try. it seems the game wont recognize the controller as an analog input, kinda like it thinks its a ps2 controller but the name still says "xbox one"

so that kinda worked but couldn't remap anything, i tried moving more xidi files over and and its the same as before but just a different name 
![image](https://github.com/user-attachments/assets/763c5f27-52ba-46d4-8c61-29a33ddbd8c7)

**Comment by emoose** [comment]
Yeah I noticed the same just now too, try changing outrun2006tweaks.ini ControllerHotPlug to false, that seemed to fix the remap page for me

**Comment by SoulLad** [comment]
> Yeah I noticed the same just now too, try changing outrun2006tweaks.ini ControllerHotPlug to false, that seemed to fix the remap page for me

![image](https://github.com/user-attachments/assets/28295344-200d-41b6-9c37-e79784a99e9f)
its less angry but still wont let me change bindings. it seems like no matter what when trying to rebind an axis option it will only read the left stick as analog, triggers seem to not be working?
[OutRun2006Tweaks.log](https://github.com/user-attachments/files/16737616/OutRun2006Tweaks.log)

**Comment by emoose** [comment]
After you press A on the axis and it asks you to press which analog to assign, does pressing & releasing the LT/RT let it register? Sometimes I have to hold it in for a few seconds and then release it for it to actually see it, not sure what's up with that.

**Comment by SoulLad** [comment]
It just progresses to the next step of calibration but does not make the confirmation noise and after pressing a trigger it still wants to only read the joystick

**Comment by emoose** [comment]
Hm maybe Xidi isn't working right for it, afaik others had issues with it too.

Maybe you could try removing Xidi's dinput8.dll and try setting it up with Steam input instead, heard some got it fixed through that before.

If you add it to steam as non-steam game you can find it in your steam library, press the controller icon on the game page, then click "enable steam input", that might be enough for it to work.

**Comment by SoulLad** [comment]
yep that worked, i play with accelerate/break axis on R2. seems to be weird about 0% input and the sensitivity bar id both delayed and only uses the bottom half of the bar. could be a frame limiter problem?
